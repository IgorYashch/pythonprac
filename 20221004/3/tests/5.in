2345.24323, 234.1
