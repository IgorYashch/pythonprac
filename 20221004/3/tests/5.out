2111.14323
